app/services/properties_creation_service.rb

```diff
@@ -0,0 +1,86 @@
+class PropertiesCreationService
+  SCHEMA_PATH = Rails.root.join('public/schemas/properties.json').to_s
+
+  def self.call(raw_post)
+    service = new(raw_post)
+    service.call
+  end
+
+  def initialize(raw_post)
+    @raw_post = raw_post
+    @payload = JSON.parse(@raw_post, symbolize_names: true)
+    @errors = []
+  end
+
+  def call
+    if json_valid? && assessment_exists?
+      return success_response if create_properties
+    end
```

Comment:
You might as well write
`return success_response if json_valid? && assessment_exists? && create_properties`
rather than an `if` inside another